### PR TITLE
Userdoc

### PR DESCRIPTION
**Pull request by MartineDeVos**
In this branch:

Modified ReadME
* Modified order of sections
* Corrected internal links
* Added scope + reading parquet files

Release + DOI (registry in Zenodo)

Added metadata + citation information

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,10 +59,10 @@ Note, these policies are broader than required for the deployment of Crunchbase.
...
 ### Installation
-Check out [this repository](https://github.com/UtrechtUniversity/ia-webscraping), and make sure you checkout the 'main' branch. Open a terminal window and navigate to the `cdx-records` directory.
+Check out [this repository](https://github.com/UtrechtUniversity/ia-webscraping), and make sure you checkout the 'main' branch. Open a terminal window and navigate to the `code` directory.
 ```bash
-# Go to CDX records folder
-$ cd cdx-records
+# Go to code folder
+$ cd code
 ```
 (The index format used by the Internet Archive is called 'cdx', hence the name)
```
Comment (maartenschermer) [inline README.md:67]: Now that we've renamed the folder 'code', this line is a little misplaced. Maybe better just remove it.

File: README.md [hunk 2]
```diff
@@ -430,6 +376,19 @@ extensions are added automatically.
...
++ job_tag: id.
++ domain: domain for which the CDX-function retrieved the URL.
++ url: full URL that was scraped.
++ page_text: full page text
++ page_links: list of page links
++ timestamp: timestamp of creation of the record
+
 
 ## Cleaning up
 ### Destroying the infrastructure
```
Comment (maartenschermer) [inline README.md:394]: Maybe 'Destroying' is a little strong; how about 'Deleting'?

File: CITATION.cff [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# YAML 1.2
+---
+authors: 
+  -
+    affiliation: "Utrecht University"
+    family-names: Schermer
+    given-names: Maarten
+    orcid:
```
[the remark this answers is not in the document]
Comment (MartineDeVos) [reply CITATION.cff:8]: Added to citation file

File: README.md (hunk 2 above)
Comment (MartineDeVos) [reply README.md:394]: You are right :-) Replaced destroying with deleting

**Pull request merged**